cross/sickchill/Makefile

```diff
@@ -1,14 +1,10 @@
 PKG_NAME = SickChill
-PKG_VERS = 2021.03.10-1
+PKG_VERS = 2021.03.28-3
 PKG_EXT = tar.gz
 PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = https://github.com/SickChill/SickChill/archive
+PKG_DIST_SITE = https://github.com/SickChill/SickChill/archive/refs/tags/
```

Comment:
@publicarray is this line wrong?

Reply:
GitHub differentiates between releases and tags. At the moment your releases are just tags: https://github.com/SickChill/SickChill/releases. (hover over tar.gz to see the url)

The last proper GitHub release was https://github.com/SickChill/SickChill/releases/tag/v2020.07.09-1 but wanted a more recent one.

Reply:
Actually now I don't know both use the `/refs/tags`